frontend/packages/dev-console/src/components/catalog/details/CatalogDetailsPanel.tsx

```diff
@@ -46,18 +46,20 @@ const CatalogDetailsPanel: React.FC<CatalogDetailsPanelProps> = ({ item }) => {
                 />
               )}
             </PropertiesSidePanel>
-            <div className="co-catalog-page__overlay-description">
-              <Stack hasGutter>
-                <SectionHeading text={t('devconsole~Description')} />
-                {!details?.descriptions?.length && description && <p>{description}</p>}
-                {details?.descriptions?.map((desc, index) => (
-                  <StackItem key={index}>
-                    {desc.label && <SectionHeading text={desc.label} />}
-                    {desc.value}
-                  </StackItem>
-                ))}
-              </Stack>
-            </div>
+            {(details?.descriptions || description) && (
```

Comment:
I will recommend that instead of adding this check `details?.descriptions` add a check for array's length `details?.descriptions?.length` or use `!_.isEmpty(details?.descriptions)` since with the current check if the array is empty and `description` is also `undefined` we will see an empty form section.